docs/deployment/index.md

```diff
@@ -0,0 +1,23 @@
+---
+html_meta:
+  "description": "Deployment of Plone"
+  "property=og:description": "Deployment of Plone"
+  "property=og:title": "Deployment of Plone"
+  "keywords": "Plone, deployment, automation"
+---
+
+(deployment-label)=
+
+# Deployment of Plone
+
+
```

Comment:
```{todo}
Intro to chapter "deployment"
```

Reply:
Fixed in c90def18b2451ca40ba1343058c8cf73539380b0